src/app/shared/services/validation/validation.service.ts

```diff
@@ -47,17 +47,11 @@ export class ValidationService {
     };
   }
 
-  validateForm(form: UntypedFormGroup): void {
-    form.updateValueAndValidity();
-
-    Object.values(form.controls).forEach((control) => {
-      control.updateValueAndValidity();
-      if ((control as UntypedFormGroup).controls) {
-        this.validateForm(control as UntypedFormGroup);
-      }
-    });
-
-    this.emitSubmit();
+  custom(options: ErrorOptions, callback: (v: any) => boolean): ValidatorFn {
```

Comment:
not clear what it validates by the name

Reply:
it is custom validator. it takes validaltion function taht is validate. the logic can be completelly different in different places. so it is ok I think. can you propose another name?

Reply:
fixed